8.x/theme-development/products.md

```diff
@@ -1,3 +1,435 @@
+
+##### EXAMPLE
+
+```php
+<div>
+   <p>{!! $product->description !!}</p> #Show product description
+</div>
+```
+
+And write in the terminal:
+
+```bash
+./deploy.sh 
+```
+
+Each product mold contains a set of attributes, for example, valves is a product whose attributes can include brand, size, and body material. To display the **attributes** of a product such as brand and body material, you can proceed as follows:
+
+##### EXAMPLE
+
+```php
+<div>
+   <hr>
+      <ul>
+         @foreach($attributes as $key_id => $attribute)
+            <li>
+               <span>{{$attribute->title}}</span> #Attribute title, body Material
+               <span>:</span>
+               @foreach($attribute->values as $val_index => $value)
+                  <span>{{$value->name}}</span>, #Attribute name, such as Cast iron metal
+               @endforeach
+            </li>
+         @endforeach
+      </ul>
+   <hr>
+<div>
+```
+
+And write in the terminal:
+
+```bash
+./deploy.sh 
+```
+
+Below you can see the full example:
+
+##### EXAMPLE
+
+```php
+<h1>{{$product->title}}</h1>
+<div>
+   <ul>
+      <li>Product code: {{$product->code}}</li> #Display product code
+      @if($product->has_discount)   #Examines the discount plan
+        <li>previous price: {{$product->previous_price}} Toman</li>
+      @endif
+      <li>product price: {{$product->latest_price}} Toman</li> #Displays the latest price
+      <li>count: {{$product->count}} Number</li> #Displays the Number
+   </ul>
+      <h4>Here you can see that what parent directories there are:</h4>
+   <ul>
+      @foreach($product->directory->getParentDirectories() as $parent_directory)
+         <li>{{$parent_directory->title}}</li> #Displays parent directories
+      @endforeach
+        <li>{{$product->title}}</li> #Product title
+   </ul>
+      <h4>Here you can see what product image there are:</h4>
+   <ul>
+      @foreach($product->images as $index=>$product_image)
+         <li>
+            <img src="{{url(ImageService::getImage($product_image, 'preview'))}}" alt="The image number {{$index}}"> #View product image
+         </li>
+      @endforeach
+   </ul>
+   <hr>
+      <ul>
+         @foreach($attributes as $key_id => $attribute)
+            <li>
+               <span>{{$attribute->title}}</span> #Attribute title, body Material
+               <span>:</span>
+               @foreach($attribute->values as $val_index => $value)
+                  <span>{{$value->name}}</span>, #Attribute name, such as Cast iron metal
+               @endforeach
+            </li>
+         @endforeach
+      </ul>
+   <hr>
+      <p>{!! $product->description !!}</p>
+</div>
+```
+
+With the help of the following command in the terminal, you can return to the default theme and see the following example in the default theme in order:
+
+```bash
+git checkout -- public/views/product-single.blade.php
+./deploy.sh 
+```
+
+**NOTE:** you are free at the front of the site and can use different modules and methods.
```

Comment:
**NOTE:** It is important to mention that you are absolutely free to manage and design any of the template pages using Larammerce methods and helpers. So please feel free to make your pages and be aware of the page naming.